openlibrary/components/MergeUI.vue

```diff
@@ -62,9 +62,14 @@ export default {
         async doMerge() {
             if (!this.$refs.mergeTable.merge) return;
             const { record: master, dupes, editions_to_move } = this.$refs.mergeTable.merge;
+            const unretrieved_editions_works = await this.$refs.mergeTable.find_unretrieved_editions;
```

Comment:
We shouldn't need to await here; `asyncComputed` does magic behind the scenes.
```suggestion
            const unretrieved_editions_works = this.$refs.mergeTable.find_unretrieved_editions;
```